sourcetracker/_plot.py

```diff
@@ -10,16 +10,272 @@
 
 import seaborn as sns
 import matplotlib.pyplot as plt
+import os
 
 
-def plot_heatmap(mpm, cm=plt.cm.viridis, xlabel='Sources', ylabel='Sinks',
-                 title='Mixing Proportions (as Fraction)'):
-    '''Make a basic mixing proportion histogram.'''
-    fig = plt.figure()
-    ax = fig.add_subplot(1, 1, 1)
-    sns.heatmap(mpm, vmin=0, vmax=1.0, cmap=cm, annot=True, linewidths=.5,
-                ax=ax)
-    ax.set_xlabel(xlabel)
-    ax.set_ylabel(ylabel)
-    ax.set_title(title)
-    return fig, ax
+class ST_graphs:
+    def __init__(self, mpm, output_dir,
+                 title='Mixing Proportions', color='viridis'):
```

Comment:
I am inclined to agree with that. However, when I set out to improve the plotting, I tried to minimize the changes I made in structure if possible. This was one of those and I haven't had problems with this yet. This is also the reason as to why I added underscores to the names instead of spaces. If you want to, or are adamant in changing this I am happy to do that. Otherwise we can keep this as is.

Reply:
If the title is a file name we should replace the space with something.

Reply:
Alright I will replace it with an underscore. Thats how I treated the file naming.

Reply:
I already did. out_file is the new file name string which replaces all spaces with underscores